package.json

```diff
@@ -50,6 +50,7 @@
     "react-native-airship": "^0.2.9",
     "react-native-keyboard-aware-scroll-view": "^0.9.5",
     "react-native-patina": "^0.1.6",
+    "react-native-webview": "^13.5.1",
```

Comment:
Because this is a native library, it needs to be a `devDependency`, and then we need to add a message in the readme informing users that we need this added to the root project.

Reply:
This still needs to be fixed.